net/HttpRequest.hpp

```diff
@@ -441,6 +462,27 @@ class Header
     /// Return true iff Transfer-Encoding is set to chunked (the last entry).
     bool getChunkedTransferEncoding() const { return _chunked; }
 
+    bool hasConnectionToken() const { return has(CONNECTION); }
+    ConnectionToken getConnectionToken() const {
+        const std::string token = get(CONTENT_LENGTH);
```

Comment:
Content-Length    = "Content-Length" ":" 1*DIGIT

Surely - we don't expect the content_length token to be anything but a number ? but I didn't read the get() impl.

Reply:
;-) .. reading your comment after my fix. Yeah: CONNECTION it is now. *slap* myself.
https://github.com/CollaboraOnline/online/commit/9998e90717a8a4083cdec3b82c9cc6d95e57634d